terraform/aws/components/gp2gp/locals_env_variables.tf

```
@@ -34,7 +34,7 @@ locals {
     },
     {
       name  = "GP2GP_MHS_INBOUND_QUEUE"
-      value = var.mhs_inbound_queue_name
+      value = var.daisy_chaining_active ? var.daisy_chaining_queue : var.mhs_inbound_queue_name
```

I think I'd actually prefer something like 
```suggestion
      value = var.gp2gp_inbound_queue_name
```

Simpler, and then if you want to use the daisy chaining queue name, you can specify it within the relevant `tfvars` file. 🤔